packages/grafana-ui/src/components/PanelChrome/PanelChrome.test.tsx

```diff
@@ -81,8 +110,38 @@ it('renders panel with a show-on-hover menu icon if prop menu', () => {
   expect(screen.getByTestId('panel-menu-button')).not.toBeVisible();
 });
 
-it.skip('renders states in the panel header if any given', () => {});
+it('renders error status in the panel header if any given', () => {
```

Comment:
Should we check if the error message is within the panel status wrapper?

Reply:
I'm not sure. The `PanelStatus` rendered inside the `panel-status-wrapper` div is its own component, we don't technically know what it is doing with the message. Then the `PanelStatus` renders a `ToolbarButton`, so for be it is responsibility of the `ToolbarButton` to have unit tests to prove that it is rendering whatever content is given to it in the expected format and place.